### PR TITLE
fix: incorrect cross chain swaps activity item values

### PR DESCRIPTION
**Pull request by infiniteflower**
## **Description**

This PR fixes the Bridge activity list item not showing the proper token symbol, token amount, and display currency amount.

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/28899?quickstart=1)

## **Manual testing steps**

1. Do a Bridge
2. Go to tx history
3. Observe proper values

### **Before**

Screenshot 2024-12-03 at 3 44 55 PM

### **After**

Screenshot 2024-12-03 at 3 43 38 PM

## **Pre-merge author checklist**

- [x] I've followed [MetaMask Contributor Docs](https://github.com/MetaMask/contributor-docs) and [MetaMask Extension Coding Standards](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [x] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [x] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [x] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: ui/hooks/useTransactionDisplayData.js [hunk 1]
```diff
@@ -383,8 +388,14 @@ export function useTransactionDisplayData(transactionGroup) {
   } else if (type === TransactionType.bridge) {
     title = t('bridgeToChain', [destChainName || '']);
     category = TransactionGroupCategory.bridge;
-    // TODO add primaryDisplayValue,primarySuffix
-    // also secondaryDisplayValue, secondarySuffix
+    // TODO also secondaryDisplayValue, secondarySuffix
+
+    primarySuffix = primaryTransaction.sourceTokenSymbol;
+    primaryDisplayValue = calcTokenAmount(
+      primaryTransaction.sourceTokenAmount,
```
Comment (micaelae) [inline ui/hooks/useTransactionDisplayData.js:395]: Does sourceTokenAmount include the MM fees?

Comment (infiniteflower) [reply ui/hooks/useTransactionDisplayData.js:395]: `sourceTokenAmount` comes from `quoteResponse.quote.srcTokenAmount`, so it does not include MM fees. What should the activity list item display? The amount that has the fees accounted for or the actual total amount that left your wallet?

Comment (micaelae) [reply ui/hooks/useTransactionDisplayData.js:395]: It should be the amount including fees, which is equal to the amount that left the user's wallet

Comment (micaelae) [reply ui/hooks/useTransactionDisplayData.js:395]: the quote's `srcTokenAmount` field only reflects the amount bridged after the MetaMask fees are taken. so `srcTokenAmount + (0.875% * srcTokenAmount)` always leaves the user's wallet

Comment (infiniteflower) [reply ui/hooks/useTransactionDisplayData.js]: Are you sure about the `quote.srcTokenAmount`? I'm looking at a quote for 0.01 UNI and it's showing `
srcTokenAmount: "10000000000000000"`

**Comment by infiniteflower** [comment]
Should look like this now 
Screenshot 2024-12-04 at 1 09 42 PM

File: ui/hooks/useTransactionDisplayData.js (hunk 1 above)
Comment (infiniteflower) [reply ui/hooks/useTransactionDisplayData.js]: Done in [02c54fb](https://github.com/MetaMask/metamask-extension/pull/28899/commits/02c54fbf1a9232d0b58f161b11d18691f61b1f6c)

File: ui/pages/bridge/types.ts [hunk 2]
```diff
@@ -57,7 +57,7 @@ export type QuoteRequest = {
   destWalletAddress?: string;
   srcChainId: ChainId;
   destChainId: ChainId;
-  srcTokenAddress: string;
+  srcTokenAddress: string; // This is the amount sent
```
Comment (micaelae) [inline ui/pages/bridge/types.ts:60]: Nit: I don't think this comment applies?

**Review by micaelae — DISMISSED** [review]

**Review by micaelae — APPROVED** [review]

**Comment by jclancy93** [comment]
I don't think this is directly related, but I'm getting an error with the validators you set up for Base tokens because the icon value is null 

image


Bridge status response - https://bridge.dev-api.cx.metamask.io/getTxStatus?bridgeId=socket&srcTxHash=0x4c57876fad21fb5149af5a58a4aba2ca9d6b212014505dd733b75667ca4f0f2b&bridge=across&srcChainId=10&destChainId=8453&refuel=false

**Review by ghgoodreau — APPROVED** [review]

**Pull request merged**